lib/helpers.ts

```diff
@@ -24,7 +24,7 @@ if (typeof window !== 'undefined') {
     win = {} as any;
 }
 
-export type DataFileID = string;
+export type synapseId = string;
```

Comment:
minor, but conventionally type names start with an uppercase letter

```suggestion
export type SynapseId = string;
```

Reply:
![image](https://github.com/gf-dcc/portal/assets/114612268/eca2f805-17f7-4f22-b650-0dfffa5c8833)
@onursumer ok yes can change! Thanks - just double checking why in this file table is the synapseId lowercase?